concourse/tasks/upload-rails-assets.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env sh
+
+set -eu
+
+mkdir -p ~/.aws
+
+cat <<EOF > ~/.aws/config
+[profile default]
+role_arn = $ASSUME_ROLE_ARN
+credential_source = Ec2InstanceMetadata
+EOF
+
+echo "Uploading rails assets from $IMAGE_ASSETS_PATH to $S3_BUCKET_PATH ..."
+
+aws s3 sync \
+  $IMAGE_ASSETS_PATH \
+  $S3_BUCKET_PATH
```

Comment:
These should be quoted to prevent word splitting (we should set up shellcheck as part of the tests for this repo 🤔)